plugins/chat/assets/stylesheets/common/chat-navbar.scss

```diff
@@ -1,23 +1,52 @@
-.chat-navbar {
-  flex-shrink: 0;
+.c-navbar {
   display: flex;
   align-items: center;
   width: 100%;
+  gap: 0.25rem;
 
   &-container {
-    padding-inline: 1rem;
-    position: sticky;
+    padding-inline: 0.5rem;
     border-bottom: 1px solid var(--primary-low);
     background: var(--secondary);
-    top: var(--header-offset);
-    height: 50px;
+    height: 46px;
+    min-height: 46px;
     box-sizing: border-box;
     display: flex;
-    z-index: z("header") - 1;
+    z-index: z("composer", "content") - 1;
+  }
+
+  .single-select-header {
+    padding: 0.3675rem 0.584rem;
+  }
+
+  .c-navbar__channel-title {
+    @include ellipsis();
+  }
+
+  .c-navbar__title {
+    @include ellipsis();
```

Comment:
```suggestion
    @include ellipsis();
    font-weight: 700;
```

Reply:
https://github.com/discourse/discourse/blob/6f3c498b83169928aba316246ddc9b5d584c5a0d/plugins/chat/assets/stylesheets/common/chat-channel-title.scss#L124-L129

We probably should add `font-weight: 700;` here too

Reply:
Yes I hesitated, maybe it's better, will add back for now :+1: